CHANGES.rst

```diff
@@ -15,14 +15,30 @@ Unreleased
   instance of :class:`.Redditor` or a string containing a redditor name and
   returns an instance of :class:`.Redditor` if the authenticated user is
   friends with the user, otherwise throws an exception.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to True on default.
+
+* Values in methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` that are left as the
+  defaults will no longer be over-written if the ``fetch`` parameter is set to
+  True, but will fill in existing values for the flair template.
+* The parameter ``text`` for methods
+  :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` is no longer required.
 
 **Removed**
 
 * Converting :class:`.APIException` to string will no longer escape unicode
   characters.
 * Module ``praw.models.modaction`` no longer exists. Pleae use the module
-  ``praw.models.mod_action`` module, or directly import ``ModAction``
+  ``praw.models.mod_action``, or directly import ``ModAction``
   from ``praw.models``.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` will no longer
+  create flairs that are using an invalid template id, but instead throw a
+  :class:`.InvalidFlairTemplateID`.
```

Comment:
As we discussed, this should be a `DeprecationWarning` for now.

Also, I prefer "raise" to "throw," as `raise` is the keyword in Python.

Reply:
I am aware, but because we are pushing to v7.0.0, I decided to implement the major breaking API changes back.